examples/dispatch.js

```diff
@@ -27,9 +27,10 @@ export default async function() {
     const contacts = page.locator('a[href="/contacts.php"]');
     await contacts.dispatchEvent("click");
 
-    const h3 = page.locator("h3");
-    const ok = await h3.textContent() == "Contact us";
-    check(ok, { "header": ok });
+    await check(page.locator('h3'), {
+      'header': async lo => lo.textContent()
```

Comment:
`async` is not needed because there's no `await`.

Reply:
So, it doesn't matter whether `textContent` returns a promise, right?

Reply:
Nope, doesn't matter.

As long as it's a promise it will be awaited by the `check` function. Non-promise values will be used as-is.

The `async` keyword is only needed if you're using `await`. Without an `await`, all it will do is make sure that the return value is wrapped in a `Promise` (basically `Promise.resolve(returnValue)`). Since we support non-promise values, having `async`

Reply:
Good to know! Thanks for explaining 🙇